sql_generators/glean_usage/templates/events_stream_v1.query.sql

```diff
@@ -74,7 +74,8 @@ WITH base AS (
         client_info.telemetry_sdk_build AS telemetry_sdk_build,
         client_info.build_date AS build_date,
         client_info.session_id AS session_id,
-        client_info.session_count AS session_count
+        client_info.session_count AS session_count,
+        client_info.windows_build_number AS windows_build_number,
```

Comment:
Can't do trailing commas in a struct
```suggestion
        client_info.windows_build_number AS windows_build_number
```